### PR TITLE
Delete a Rollup from the Rollup View in the custom UI

### PR DESCRIPTION
**Pull request by bethbrains**
# Critical Changes

# Changes

- Admin can delete a Rollup from the Rollup View in the custom CRLP UI

# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: src/aura/CRLP_Rollup/CRLP_RollupController.js [hunk 1]
```diff
@@ -110,11 +113,16 @@
      */
     onSave: function(cmp, event, helper){
         var activeRollup = cmp.get("v.activeRollup");
-        var canSave = helper.validateFields(cmp);
-        if(canSave){
+        if (cmp.get("v.mode") == 'delete') {
```
Comment (force2b) [inline src/aura/CRLP_Rollup/CRLP_RollupController.js]: I may have put a comment here already, but if not: `==` in Javascript should be `===`.

File: src/aura/CRLP_Rollup/CRLP_RollupController.js [hunk 2]
```diff
@@ -110,11 +113,16 @@
...
+            helper.toggleModal(cmp);
             helper.saveRollup(cmp, activeRollup);
-            cmp.set("v.mode", 'view');
-            helper.updateRollupName(cmp);
+        } else {
```
Comment (force2b) [inline src/aura/CRLP_Rollup/CRLP_RollupController.js]: Does it make sense to combine these 3 lines into one?
`} else if (helper.validateFields(cmp) === true) {`

File: src/classes/CRLP_RollupUI_SVC.cls [hunk 3]
```diff
@@ -603,12 +603,17 @@ public with sharing class CRLP_RollupUI_SVC {
                         if (objectType == 'Rollup') {
                             Id rollupId = CRLP_Rollup_SEL.getRollupIdByDeveloperName(recordName);
                             response.completed = true;
-                            response.rollupItem = createRollupItem(CRLP_Rollup_SEL.getRollupById(rollupId));
+                            if (mode != 'delete') {
```
Comment (force2b) [inline src/classes/CRLP_RollupUI_SVC.cls]: Inequality operations should be `!==`

Comment (bethbrains) [reply src/classes/CRLP_RollupUI_SVC.cls]: as per quip convo, this one is apex so it's fine.

**Pull request merged**